Source/Scene/Scene.js

```diff
@@ -213,8 +217,32 @@ define([
             canvas.parentNode.appendChild(creditContainer);
         }
 
+        if (!defined(leftCreditContainer) || !defined(rightCreditContainer)) {
```

Comment:
This (and `creditContainer` initialization) is kind of bulky here; can we create a static helper function on `CreditDisplay`?